sqlglot/dialects/snowflake.py

```diff
@@ -317,6 +317,8 @@ class Parser(parser.Parser):
             "TO_VARCHAR": exp.ToChar.from_arg_list,
             "ZEROIFNULL": _build_if_from_zeroifnull,
         }
+        FUNCTIONS.pop("IFNULL")
```

Comment:
does oracle also support this? should we do the same for it?

Reply:
Can't find it in the docs